aca_preview/preview.py

```diff
@@ -126,6 +134,24 @@ def get_summary_text(acas):
 
 
 class ACAReviewTable(ACATable):
+    def set_stars_and_mask(self):
+        """Set stars attribute for plotting.
+
+        This includes compatibility code to deal with somewhat-broken behavior
+        in 4.3.x where the base plot method is hard-coded to look at
+        ``acqs.stars`` and ``acqs.bad_stars``.
+
+        """
+        # Get stars from AGASC and set ``stars`` attribute
+        self.set_stars()
```

Comment:
This looks good but not sure if you want to handle the different behaviors in set_stars with regard to filter_near_fov.  Is there a use case where it would make a difference?

Reply:
I can't think of any reason to care in this context.  If we come across a driver then we'll address it then.